apps/docs/src/app/core/component-docs/select/select-docs.component.html

```diff
@@ -1,107 +1,130 @@
-<fd-docs-section-title [id]="'standard'" [componentName]="'select'">
-    Standard Select
+<fd-docs-section-title [id]="'mode'" [componentName]="'select'">
+    Select mMdes
 </fd-docs-section-title>
-<description>A basic example of the select component with no advanced features.</description>
+<description>
+    The Select control supports a wide variety of modes to display your list items. <br/>
+    
+</description>
 <component-example>
-    <fd-select-basic-example></fd-select-basic-example>
+    <fd-select-mode-example></fd-select-mode-example>
 </component-example>
-<code-example [exampleFiles]="selectBasic"></code-example>
+<code-example [exampleFiles]="selectMode"></code-example>
 
 <separator></separator>
 
-
-<fd-docs-section-title [id]="'Types'" [componentName]="'select'">
-    Select Border Types
+<fd-docs-section-title [id]="'semantic'" [componentName]="'select'">
+    Select Semantic States
 </fd-docs-section-title>
-<description>An example of the select component with diffrent types of Border type and ability to add icon.
-</description>
+<description></description>
```

Comment:
The states of the Select control can be customized by assigning different border colors for each state.

@Michal: The description was removed here but I think it would be good to add one because all the other examples do :)